### ISSUE TITLE
Limit search results by Molecular Weight

### ISSUE THREAD

**Issue by and-tos** [Actelion/openchemlib-hyperspace#5]
Is it possible to limit the search by global additive properties such as molecular weight?

**Comment by lithom** [comment]
Yes, it is possible to do this efficiently. The algorithm first computes the "combinatorial hits", i.e. all synthon combinations that match. Then one could compute these additive properties for the synthons, and then when looping over all synthon combinations one could assemble only the combinations that are within the specified range. I will consider to include this in some way in the GUI.
